tests/api_test/api_test.cpp

```diff
@@ -1082,68 +1082,6 @@ test_sock_addr_native_program_load_attach(const char* file_name)
 
 DECLARE_REGRESSION_TEST_CASE("0.11.0")
 
-// The below tests try to load native drivers for invalid programs (that will fail verification).
```

Comment:
I added back the netsh tests (they don't need bpf2c), and I've now removed the `_load_invalid_program` tests in api_test.cpp.

They are specifically to test that programs compiled with noverify fail to load, and noverify doesn't exist anymore.

@Alan-Jowett Should we preserve these tests in some way (e.g. empty.sys for when that is eventually supported), or just delete them, since we already test for programs failing verification and have the issue for supporting empty programs?

Reply:
Yes, please go ahead and remove them.